server/models/schema.sql

```diff
@@ -1177,6 +1177,7 @@ CREATE TABLE `patient` (
   `notes`                TEXT,
   `middle_name`          VARCHAR(150),
   `hospital_no`          VARCHAR(150),
+  `picture_patient`      VARCHAR(150),
```

Comment:
A better name for the field is either "image" or "avatar".  Could you rename this column?